riscv-runtime/powdr.x

```diff
@@ -1,40 +1,70 @@
 # Powdr linker script.
 #
-# If you are using powdr-riscv-runtime, it expects the following
+# A segment of type LOOS+0xda (0x600000da) is understood by powdr, and it
+# delimits an unconstrained memory region that can be used to pass data
+# to the ZK program. If present, this region must be bigger than
```

Comment:
```suggestion
# to the guest program. If present, this region must be bigger than
```

Reply:
I think it is fine the way it is.

Reply:
I think `guest` is more accurate in the sense of a VM, but fine